backend/src/main/java/com/exadel/carinsurance/mapper/AssignmentMapper.java

```diff
@@ -0,0 +1,56 @@
+package com.exadel.carinsurance.mapper;
+
+import com.exadel.carinsurance.model.assignment.AssignmentEntity;
+import com.exadel.carinsurance.model.assignment.ContactInfoEntity;
+import com.exadel.carinsurance.model.request.AssignmentRequestEntity;
+import com.exadel.carinsurance.model.response.AssignmentResponseEntity;
+import com.exadel.carinsurance.model.response.ContactInfoResponseEntity;
+import com.exadel.carinsurance.model.response.DirectionOfImpactResponseEntity;
+import com.exadel.carinsurance.model.response.VehicleConditionInfoResponseEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AssignmentMapper {
+  public static AssignmentEntity mapToAssignment( AssignmentRequestEntity assignmentRequest ) {
+    return AssignmentEntity
+        .builder()
+        .dateOfIncident( assignmentRequest.getDateOfIncident() )
+        .build();
+  }
+
+  public static AssignmentResponseEntity mapToAssignmentResponse( AssignmentEntity assignment ) {

```

Comment:
Please don't use static methods for mapper. By using static methods for such operations you will have high coupling between your components. This is also much less testable as classes that will use such menhods will not have any easy ways to mock their behavior